v3/src/components/axis/components/axis.scss

```diff
@@ -1,60 +1,64 @@
-.axis {
-  background-color: white;
+.graph-plot {
```

Comment:
Wrapping these rules inside `.graph-plot` means they only apply to the graph and won't be applied to the slider. Was that the goal? If so, they should probably be in the graph folder rather than the axis folder.

Reply:
Moved these to graph.scss